apps/browser/qml/pages/components/HistoryContextMenu.qml

```diff
@@ -0,0 +1,45 @@
+/****************************************************************************
+**
+** Copyright (c) 2014 - 2021 Jolla Ltd.
+** Copyright (c) 2021 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.1
+import Sailfish.Silica 1.0
+
+Component {
+    ContextMenu {
+        MenuItem {
+            //% "Open in private tab"
+            text: qsTrId("sailfish_browser-me-open-in-private-tab")
+            onClicked: view.openInPrivateTab(model.url, model.title)
```

Comment:
When the user is already in Private mode, selecting the item will open it in private mode. In this case, perhaps the menu could have "Open in normal mode" instead, to make it convenient for the user to switch out of private mode?

Reply:
Done